### ISSUE TITLE
You have an "out of sync" message in version 3.5.9.9 (but your blockchain is actually uptodate and fine) ?

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/GRC#30]
Do you see in your wallet an "out of sync" text at the bottom AND you are running version [3.5.9.9](https://steemit.com/gridcoin/@erkan/new-release-gridcoin-version-3-5-9-9-leisure-not-mandatory) ?

<br>

It seems there is this bug:
* "the wallet miner message in the client _doesn't clear the 'out of sync' message_"

you see this at the bottom with like:
* "Stake Weight ...;  Searching for coins **Out of Sync**"

<br>

No worries, as long as you have: 
* a green check mark and 
* the green up-arrow (see pic), 

you should be fine

![green_aug8](https://user-images.githubusercontent.com/642286/29185527-b67d5ac4-7e09-11e7-94c3-3f86ccbb7c78.png)

**Comment by Erkan-Yilmaz** [comment]
@Foggyx420 you reported this originally... should I add it to the bugtracker, or is it already "fixed" and to come in future version?

**Comment by iFoggz** [comment]
Already delt with I believe.

**Comment by Erkan-Yilmaz** [comment]
yes, see [here](https://github.com/gridcoin/Gridcoin-Research/issues/505#issuecomment-321742714) (not sure when the V8 release will come, though)

**Comment by Erkan-Yilmaz** [comment]
tagged with: [process](https://github.com/Erkan-Yilmaz/GRC/labels/process)

bug on main UI everyone will see
* damage: irritates/confuses users

risk, simplified = damage * frequency

not released in like 2 weeks

**Comment by tomasbrod** [comment]
To my knowledge, it was not fixed yet.
It is caused by new and old code fighting over the "Client message". This is only a UI issue.
